### PR TITLE
[DEV APPROVED] Finish off multiple offices work

### PR DESCRIPTION
**Pull request by JiggyPete**
Snagging list around multiple offices functionality. (Each firm can have multiple offices.)

Update schema, tests and make use of https://github.com/moneyadviceservice/mas-rad_core/pull/95

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -174,25 +180,25 @@ GEM
...
       rack (>= 1.0)
-    rails (4.2.1)
-      actionmailer (= 4.2.1)
-      actionpack (= 4.2.1)
-      actionview (= 4.2.1)
-      activejob (= 4.2.1)
-      activemodel (= 4.2.1)
-      activerecord (= 4.2.1)
-      activesupport (= 4.2.1)
+    rails (4.2.4)
```
Comment (neoeno) [inline Gemfile.lock]: Did you mean to upgrade rails? If so, should probably go in its own commit

File: app/helpers/self_service/self_service_helper.rb [hunk 2]
```diff
@@ -56,8 +56,8 @@ def first_registered_firm_for(principal)
 
     def firm_language_select(value, html_options = {})
       options = options_from_collection_for_select(
-        Firm.available_languages,
-        :iso_639_1,
+        Languages::AVAILABLE_LANGUAGES,
+        :iso_639_3,
```
Comment (neoeno) [inline app/helpers/self_service/self_service_helper.rb]: I don't think this belongs here

File: spec/features/self_service/firms_edit_spec.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@
                       ethical_investing_flag: true,
                       sharia_investing_flag: true,
                       status: :restricted,
-                      languages: ['fr'])
+                      languages: ['fra'])
```
Comment (neoeno) [inline spec/features/self_service/firms_edit_spec.rb]: Or this. You get the idae

**Comment by neoeno** [comment]
This commits seems wrong: https://github.com/moneyadviceservice/rad/commit/0f038212bcfb78352f4b1d08c8c24c3797a31e8f — all these changes should be in master. See here: https://github.com/moneyadviceservice/rad/blob/master/app/helpers/self_service/self_service_helper.rb#L59-L60

**Comment by JiggyPete** [comment]
@neoeno fixed with rebase from master.

**Comment by asmega** [comment]
:+1:

**Comment by tomas-stefano** [comment]
:+1:
